Tests/TestCombine/FixtureCombine.swift

```diff
@@ -0,0 +1,194 @@
+ import MockoloFramework
+
+ let combineProtocol = """
+/// \(String.mockAnnotation)(subject: dictionaryPublisher = CurrentValueSubject; myPublisher = PassthroughSubject; noDefaultSubjectValue = CurrentValueSubject)
```

Comment:
A better parameter name might be "combine" to be distinguished from "rx" which also allows Subjects.  The same param can be used to denote @Published as well, like so: 

`@mockable(combine: foo = CurrentValueSubject; bar = @Published)`